### PR TITLE
React native: 6.1.5 (squashed)

### PR DESCRIPTION
**Pull request by chrisnojima**
- [x] hot
- [x] cold
- [x] dev ios
- [x] dev android
- [x] deps
- [x] story
- [x] rn story
- [x] prod
- [x] prod ios
- [x] prod android

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
         command: yarn install --pure-lockfile --ignore-engines --network-concurrency 1
     - run:
         working_directory: shared
-        command: yarn global add react-native-cli
+        command: yarn global add react-native-cli npm
```
Comment (songgao) [inline .circleci/config.yml]: What's the purpose of `npm` here?

File: shared/android/build.gradle [hunk 2]
```diff
@@ -77,5 +77,6 @@ allprojects {
 
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```
Comment (songgao) [inline shared/android/build.gradle]: What's this for?

File: .circleci/config.yml (hunk 1 above)
Comment (chrisnojima) [reply .circleci/config.yml]: the rn cli uses npx

File: shared/android/build.gradle (hunk 2 above)
Comment (chrisnojima) [reply shared/android/build.gradle]: this is based on the shipping template and we already had this before

**Pull request merged**